.stylelintrc

```diff
@@ -41,7 +41,7 @@
             }
         ],
         "scale-unlimited/declaration-strict-value": [
-            ["/color/", "fill", "stroke", "z-index", "font-size"],
+            ["/color/", "fill", "stroke", "font-size"],
```

Comment:
removing this from the stylelint definitions until we have a strategry around z indexing